Monitor Enhancements release notes (#4093)
SumoLogic/sumologic-documentation: blog-service/2024-04-16-apps.md

---

> title: Monitor Enhancements (Apps)

Reviewer: ```suggestion
title: Enhancements to Terraform-Based Monitors (Apps)
```

image: https://help.sumologic.com/img/sumo-square.png
keywords:
  - apps
  - monitors
  - terraform
hide_table_of_contents: true
authors:
  - url: https://help.sumologic.com/release-notes-service/rss.xml
    image_url: /img/release-notes/rss-orange.png
---

> We're excited to announce an update of all our terraform based monitors. Below are the list of enhancements that were a part of this update.

Reviewer 2: ```suggestion
We're excited to announce an update of all our Terraform based monitors. Below are the list of enhancements that were a part of this update.
```

> - Updated Kubernetes monitors to make it compatible with new helm chart versions.
> - Upgraded AWS and Sumologic provider versions for CVE fixes.

Reviewer: I have a couple edits on top of John's

```suggestion
- Updated Kubernetes monitors for compatibility with new Helm chart versions.
- Upgraded AWS and Sumo Logic provider versions to address CVE fixes.
```
